bosh-setup/azuredeploy.json

```diff
@@ -89,19 +66,42 @@
     },
     "enableDNSOnDevbox": {
       "type": "bool",
+      "defaultValue": true,
       "allowedValues": [
         true,
         false
       ],
       "metadata": {
-        "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+        "description": "A default DNS will be setup in the devbox if it is true."
+      }
+    },
+    "tenantID": {
+      "type": "string",
+      "defaultValue": "TENANT-ID",
```

Comment:
@singhkay 
The default values in azuredeploy.json works for a successful deployment. But customers need to specify the other parameters in azuredeploy.parameters.json. Because the storage account name needs to be unique and the password has some requirements just as @template-deploy-bot said below.

Should I change azuredeploy.parameters.json and let @template-deploy-bot test again?
Thanks